.github/workflows/release-cli.yml

```diff
@@ -70,7 +70,7 @@ jobs:
             CMD_BUILD: PyInstaller -F -c -n dds_cli_win_x86_64.exe -i resources/scilifelab.ico  --exclude-module=tests --log-level INFO dds_cli/__init__.py
             OUT_FILE_NAME: dds_cli_win_x86_64.exe
             ASSET_MIME: application/vnd.microsoft.portable-executable
-          - os: ubuntu-latest
+          - os: ubuntu-22.04, ubuntu-20.04
```

Comment:
I think we need one for each right? Otherwise both will have the same OUT_FILE_NAME below

Reply:
you mean one separate `- os:` block for each, yes that makes sense

Reply:
Yeah, thought of that too. Go with `ubuntu-latest` and `ubuntu-20.04`

Reply:
And add a comment by `ubuntu-latest` that it's `22.04` at the moment of writing